samples/.eslintrc

```diff
@@ -10,6 +10,7 @@ globals:
     CallAnalyzer: true
     Dashboards: true
     DataGrid: true
+    Grid: true
```

Comment:
I believe `DataGrid` can be removed now, right?

Reply:
Not yet, because in Dashboards we're still using DataGrid (which will be converted to Grid Pro in the closest future).